doc/developer/platform/formalism.md

```diff
@@ -194,3 +199,111 @@ A view introduces *constraints* on the capabilities and frontiers of its input a
     Outputs may need to be recovered in the case of failure, but also in other less dramatic query migration scenarios.
 
 These constraints are effected using the capability abstractions.
+The Compute layer acquires and maintains read capabilities for collections managed by it, and by the Storage layer.
+
+# Adapter
```

Comment:
Is the adapter what we currently call the coordinator?

Reply:
SQL + coordinator, yeah! It got updated here: https://github.com/MaterializeInc/materialize/commit/d4ba672d76fc8a8b599580c32c2574f6d6f836b7